.travis.yml

```diff
@@ -1,15 +1,33 @@
-language: haskell
+# https://docs.haskellstack.org/en/stable/travis_ci/
```

Comment:
At the moment, it's the only way I'm able to build this project at my laptop. `nix` infrastructure takes too much effort to maintain, especially with non-linux OS.
At the same time, even if `nix` works in Travis, it cannot build this particular project in a given time quota.

Anyway, let's give `stack` a chance.